tower-grpc-interop/Dockerfile-server-go

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.9
+
+ENV PACKAGE google.golang.org/grpc/interop/server
+ENV PACKAGEDIR $GOPATH/src/$PACKAGE
+
+WORKDIR /
+RUN go get -u $PACKAGE
+WORKDIR $PACKAGEDIR
+
+RUN ["go", "install", "."]
+ENTRYPOINT ["server"]
```

Comment:
If this docker image can be published (which i think it should be, because i don't think it makes much sense to rebuild on every CI/etc), then we should make this a multistage build so that the resulting image only includes the binary and not the full golang image.

Reply:
Addressed in 933a6875d4fc700ac25e35e834291915c221a5cf.